src/components/show-more-collapse/ShowMoreCollapse.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Box } from '@mui/material'
+import Typography from '@mui/material/Typography'
+import Collapse from '@mui/material/Collapse'
+
+import { styles } from '~/components/show-more-collapse/ShowMoreCollapse.styles'
+
+interface ShowMoreCollapseProps {
+  title: string
+  description: string
+  collapsedSize?: number
+  collapsedTextLength?: number
+}
+
+const ShowMoreCollapse: FC<ShowMoreCollapseProps> = ({
+  title,
+  description,
+  collapsedSize = 70,
+  collapsedTextLength = 390,
+  ...props
+}) => {
+  const { t } = useTranslation()
+  const [expanded, setExpanded] = useState<boolean>(false)
+
+  const handleChange = () => {
+    setExpanded((prev) => !prev)
+  }
+
+  const hasMoreText = description.length > collapsedTextLength
+  const collapsedHeight =
+    (description.length / collapsedTextLength) * collapsedSize
+
+  const collapsedDescription = (
+    <Typography sx={styles.description(collapsedHeight)}>
+      {expanded ? description : description.slice(0, collapsedTextLength)}
+    </Typography>
+  )
+
+  return (
+    <Box>
+      <Typography sx={styles.title}>{title}</Typography>
+
+      <Collapse collapsedSize={collapsedSize} in={expanded} {...props}>
```

Comment:
if you want to add here props that Collapse component from MUI receives, so it i think it needs to add extending of CollapseProps interface from MUI to ShowMoreCollapseProps

Reply:
you are right, fixed